public/js/i18n.js

```diff
@@ -58,89 +58,91 @@ function switchLanguage(lang) {
     fileName = url_list[url_list.length - 1];
     //default file is index
     if (fileName == "") {
-	fileName = "index";
-	cur_url = cur_url + "index";
+        fileName = "index";
+        cur_url = cur_url + "index";
     }
     //check if the current page is of Chinese version
     var flag = fileName.lastIndexOf("_zh");
     if (lang === "English") {
-	createCookie(LOCAL_LANG_COOKIE, "English");
-	if ( flag > 0 ){
+        $('title').text('Crosswalk - build world class hybrid apps');
```

Comment:
yeah this is the only line of change, it provides the default title for each page, and will be overwritten by the loading of .md file for each page. Anyway this may not be a good solution here. We can hold this patch until we find a better solution.
I'll only submit the first patch instead.